### ISSUE TITLE
lines_and_planes/two_planes_coincide

### ISSUE THREAD

**Issue by ahensbergen** [PRIME-TU-Delft/Open-LA-Applets#120]
### What needs to change?

DESTROY
Voegt na figuur 1.28    (Two planes without a common point in common;  weird titel trouwens)  NIKS TOE.

Aldus R & A

**Comment by yustarandomname** [comment]
This image is referred to in the text. This image (fig. 1.30) is written about like this:
> Two planes in 
 can never have a single point of intersection. They can, however, have infinitely many common points. This can occur when the two planes have a line of intersection, as we can see in Figure 1.29. On the other hand, it is also possible that the two planes coincide, see **Figure 1.30**. In this case, each point on one of the two planes is an intersection point.

In short, first, the text needs a rewrite before this image can be removed.
